src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -38,9 +38,9 @@ There is no separate "routing config" to maintain in an Astro project! When you
 
 An Astro page file can specify dynamic route parameters in its filename to generate multiple, matching pages. For example, `src/pages/authors/[author].astro` generates a bio page for every author on your blog. `author` becomes a _parameter_ that you can access from inside the page.
 
-In Astro's default static output mode, these pages are generated at build time, and so you must predetermine the list of `author`s that get a corresponding file. In SSR mode, a page will be generated on request for any route that matches.
+In Astro's default `static` output mode, these pages are generated at build time, and so you must predetermine the list of `author`s that get a corresponding file. In `server` or `hybrid` mode, a page will be generated on request for any route that matches.
```

Comment:
This section is specifically calling out on-demand rendering so conflating “server” and “hybrid” is a little confusing IMO.